keel-api/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -52,9 +53,9 @@ class DefaultConfiguration {
   fun noResourcePlugins(): List<ResolvableResourceHandler<*, *>> = emptyList()
 
   @Bean
```

Comment:
`csrfDisable()` is not needed and was making it so that fiat never got a chance to handle requests.
`authenticatedRequestFilter()` adds the Spinnaker filter we need to correctly parse the headers like `X-SPINNAKER-USER` and add them to the `mdc`.